<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dgMstBusqueda</class>
 <widget class="QDialog" name="dgMstBusqueda">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>633</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vl">
    <item>
     <widget class="QDockWidget" name="dw">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="features">
       <set>QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <widget class="QWidget" name="dockWidgetContents"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbAceptar">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancelar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>aceptar()</slot>
 </slots>
</ui>
